openapi.yml

```diff
@@ -143,7 +143,7 @@ paths:
           example: 'druid:bc123df4567'
           schema:
             $ref: '#/components/schemas/Druid'
-  /v1/objects/{id}/checksum.json:
+  /v1/objects/{id}/checksum:
```

Comment:
It looks like there are other endpoints that end in .json ?

Reply:
Took a closer look. I think this is the only one that ends in `.json`. However, there is some odd use of `.json` appended to druids. Not sure why it is done that way.